pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/mercury/MercuryOrderDao.java

```diff
@@ -197,6 +197,49 @@ public int create(MercuryOrderDto mercuryOrderDto) {
         throw new IllegalStateException("Call the version of this method that includes the raw json payload");
     }
 
+    public int create(MercuryOrderDto mercuryOrderDto, String messageJson) {
+        SimpleResult results = inTransaction(conn -> {
+            SimpleResult execResult = new SimpleResult();
+            try (PreparedStatement stmt = conn.prepareStatement(SQL_INSERT_MERCURY_ORDER, Statement.RETURN_GENERATED_KEYS)) {
+                stmt.setString(1, mercuryOrderDto.getOrderId());
+                stmt.setLong(2, mercuryOrderDto.getOrderDate());
+                stmt.setString(3, mercuryOrderDto.getDdpParticipantId());
+                stmt.setInt(4, mercuryOrderDto.getKitTypeId());
+                stmt.setString(5, mercuryOrderDto.getBarcode());
+                stmt.setInt(6, mercuryOrderDto.getDdpInstanceId());
+                stmt.setString(7, mercuryOrderDto.getCreatedBy().orElse(""));
+                stmt.setString(10, messageJson);
+                stmt.setNull(11, Types.VARCHAR);
+                if (mercuryOrderDto.getTissueId() != null) {
+                    stmt.setString(8, String.valueOf(mercuryOrderDto.getTissueId()));
+                } else {
+                    stmt.setNull(8, Types.INTEGER);
+                }
+                if (mercuryOrderDto.getDsmKitRequestId() != null) {
+                    stmt.setString(9, String.valueOf(mercuryOrderDto.getDsmKitRequestId()));
+                } else {
+                    stmt.setNull(9, Types.INTEGER);
+                }
+
+                stmt.executeUpdate();
+                try (ResultSet rs = stmt.getGeneratedKeys()) {
+                    if (rs.next()) {
+                        execResult.resultValue = rs.getInt(1);
+                    }
+                }
+            } catch (SQLException ex) {
+                execResult.resultException = ex;
+            }
+            return execResult;
+
+        });
+        if (results.resultException != null) {
+            throw new DsmInternalError(String.format("Error inserting mercury order for order with json {}", messageJson),
+                    results.resultException);
+        }
+        return (int) results.resultValue;
+    }
+
```

Comment:
checkstyle was really unhappy about the two `create` methods not being next to each other